app/scripts/models/user.js

```diff
@@ -626,6 +627,17 @@ define(function (require, exports, module) {
           }
           return exists;
         });
+    },
+
+    /**
+     * Report/reject the unblockCode for the given account
```

Comment:
This was a little confusing to me. I read it as either report or reject. Should it be reject and report? From looking at the [auth-server endpoint](https://github.com/mozilla/fxa-auth-server/blob/2c0561df2aae7d166a21c05b70aaaff6efe982e8/lib/routes/account.js#L1679), it appears to log an error, is this what is meant by report?